README.md

```diff
@@ -94,7 +94,7 @@ The following default options will be mixed in with your global `auth` object fr
     secure: true // whether cookies should only be available over HTTPS
   },
   jwt: {
-    header: { typ: 'access' }, // by default is an access token but can be any type
+    header: { type: 'access' }, // by default is an access token but can be any type
```

Comment:
Nope it is supposed to be `typ`. However, this reminded me that the default value really should be `JWT`